crates/optimism/payload/src/builder.rs

```diff
@@ -156,7 +150,7 @@ where
 /// a result indicating success with the payload or an error in case of failure.
 #[inline]
 pub(crate) fn optimism_payload<EvmConfig, Pool, Client>(
-    evm_config: EvmConfig,
+    evm_config: &EvmConfig,
```

Comment:
this is not strictly required
```suggestion
    evm_config: EvmConfig,
```

Reply:
EvmConfig is auto_impl for & so this shouldn't be required by we can also do &

Reply:
Wouldn't work :sweat_smile:.

```
error[E0521]: borrowed data escapes outside of method
   --> crates/optimism/payload/src/builder.rs:108:9
    |
104 |         &self,
    |         -----
    |         |
    |         `self` is a reference that is only valid in the method body
    |         let's call the lifetime of this reference `'1`
...
108 |         optimism_payload(&self.evm_config, args, cfg_env, block_env, self.compute_pending_block)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         `self` escapes the method body here
    |         argument requires that `'1` must outlive `'static`
```